VideoCollection.js

```diff
@@ -0,0 +1,29 @@
+VideoCollection = Backbone.Collection.extend({
+  // url is currently hard-coded with search value 'cats'
+  url: 'https://www.googleapis.com/youtube/v3/search?key=AIzaSyDTEoq5bStax1IZYW9UZxabnL1k8kwUpC8&part=snippet&type=video&q=cats',
+
+
+  // urlSearchAppender: function (search) {
+  //    this.url = this.url + search
+  //    console.log('input string appended to url')
+  // },
+
+
+  model: VideoModel,
+
+  parse: function (response) {
+    var videoObj = [];
+
+    for (var i = 0; i < response.items.length; i++) {
```

Comment:
Try to use the forEach or map/filter type functions rather than these loops unless you really need the index value.